crates/pink-libs/subrpc/src/lib.rs

```diff
@@ -220,14 +220,17 @@ pub fn create_transaction_ext<T: Encode>(
     call_data: UnsignedExtrinsic<T>,
     era: Era,
     tip: u128,
+    mode: u8,
+    metadata_hash: Option<[u8; 32]>,
 ) -> core::result::Result<Vec<u8>, Error> {
     let additional_params = (
         spec_version,
         transaction_version,
         genesis_hash,
         era_checkpoint,
+        metadata_hash,
     );
-    let extra = (era, Compact(nonce), Compact(tip));
+    let extra = (era, Compact(nonce), Compact(tip), mode);
```

Comment:
Is this backward compatible?

Reply:
The current version doesn't work anymore with the polkadot-felowship upgrade. To check, you can run the test with --ignored and --test-threads=1 options. The tests will fail.
This pr fixes the issue.

Reply:
> So it is not backward compatible with the previous nodes but I assume all nodes will be upgraded or are already upgraded (Shiden and Astar have been upgraded).

Sounds OK to me.